modules/salt/pages/config-modules.adoc

```diff
@@ -0,0 +1,39 @@
+[[config-modules]]
+= Configuration Modules
+
+[IMPORTANT]
+====
+This feature is a technology preview.
+====
+
+You can use Salt execution and state modules to configure your {productname} Server.
```

Comment:
Hmm, not sure if this is the best wording.

So these modules are called "Uyuni configuration modules" and they can target both Uyuni and SUSE Manager Servers.

As it is, this sentence seems to refer to generic Salt execution and state modules: of course one could use them to do whatever, including Server configuration, but here we are presenting some specific new modules that are tailored for that scope.

I'd still call them Uyuni configuration modules (just as the package name) and clarify they also apply to SUSE Manager contexts.

Reply:
OK, that's what had me confused! I've updated the wording now, so hopefully it's clearer.